app/models/agents/ftpsite_agent.rb

```diff
@@ -176,17 +182,25 @@ def saving_entries
       new_files.sort_by { |filename|
         found_entries[filename]
       }.each { |filename|
-        create_event payload: {
+        create_event payload: get_file_pointer(filename).merge({
           'url' => (base_uri + uri_path_escape(filename)).to_s,
           'filename' => filename,
           'timestamp' => found_entries[filename],
-        }
+        })
       }
 
       memory['known_entries'] = found_entries
       save!
     end
 
+    def get_io(file)
```

Comment:
True, but that is already possible with the WebsiteAgent with a decent connection and the default 2 minute job timeout you can load about 1.2GB. Not sure if we can protect Huginn from harmful users.